Observer/OrderSaveAfter.php

```diff
@@ -30,13 +30,19 @@ function execute(Observer $ob) {
 				if ($cfg->getValue('stock2shop/order_export/enable', SS::SCOPE_STORE, $o->getStore())) {
 					/** @var string $state */ /** @var string $status */
 					list($state, $status) = [$o->getState(), $o->getStatus()];
-					try {$res = self::post(Payload::get($o), $o->getStore());} /** @var string $res */
-					catch (\Exception $e) {$res = $e->getMessage();}
+					try {
+						$payload = Payload::get($o);
+						$res = self::post($payload, $o->getStore());
+					} /** @var string $res */
+					catch (\Exception $e) {
+						$res = $e->getMessage();
+					}
 					$h = $o->addStatusHistoryComment(__(
 						implode('<br>', [
 							"The Stock2Shop's webhook is notified."
 							,"The order's status: «<b>{$status}</b>»."
 							,"The order's state: «<b>{$state}</b>»."
+							,sprintf("The serialized payload: %s", serialize($payload))
```

Comment:
Let's maybe change this so that it only does this debugging serialize if there was an exception?

We should also probably run it through an html escaping function (in case there is some html content in there) - same for the webhook's response

Reply:
Done